### PR TITLE
Brands & suboptions in Platform portal

### PR DESCRIPTION
**Pull request by rbatistadev**
#### Type Of Change 
- [ ] Small bug fix
- [X] New feature or enhancement
- [ ] Breaking change

#### Checklist:
- [X] Commits are named and have tag following [commit rules](https://github.com/irontec/ivozprovider/blob/bleeding/doc/dev/en/commits.md)
- [X] Commits are split per component (schema, web/admin, kamusers, agis, ..)
- [X] Changes have been tested locally
- [ ] Fixes an existing issue (Fixes #XXXX) 
- [ ] Upport from existing Pull request #XXXX

### CODE REVIEW

File: web/portal/platform/src/entities/Currency/Currency.tsx [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  },
+};
+
+const Currency: EntityInterface = {
+  ...defaultEntityBehavior,
+  icon: AccountTreeIcon,
+  iden: 'Currency',
+  title: _('Currency', { count: 2 }),
+  path: '/currencies',
+  toStr: (row: any) => row.id,
```
Comment (fmendozzac) [inline web/portal/platform/src/entities/Currency/Currency.tsx:32]: Maybe 'id' is not the best toStr

Comment (fmendozzac) [reply web/portal/platform/src/entities/Currency/Currency.tsx:32]: any is forbidden

File: web/portal/platform/src/entities/Brand/ForeignKeyGetter.tsx [hunk 2]
```diff
@@ -0,0 +1,22 @@
+import { autoSelectOptions } from '@irontec/ivoz-ui/entities/DefaultEntityBehavior';
+import { ForeignKeyGetterType } from '@irontec/ivoz-ui/entities/EntityInterface';
+import { BrandPropertyList } from './BrandProperties';
+
+/** TODO remove this file unless you need to change default behaviour **/
```
Comment (fmendozzac) [inline web/portal/platform/src/entities/Brand/ForeignKeyGetter.tsx:5]: Remove this file

File: web/portal/platform/src/entities/Brand/ForeignKeyResolver.tsx [hunk 3]
```diff
@@ -0,0 +1,22 @@
+import { autoForeignKeyResolver } from '@irontec/ivoz-ui/entities/DefaultEntityBehavior';
+import { foreignKeyResolverType } from '@irontec/ivoz-ui/entities/EntityInterface';
+import { BrandPropertiesList } from './BrandProperties';
+
+/** TODO remove this file unless you need to change default behaviour **/
```
Comment (fmendozzac) [inline web/portal/platform/src/entities/Brand/ForeignKeyResolver.tsx:5]: Remove this file

File: web/portal/platform/src/entities/Brand/SelectOptions.ts [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+  callback,
+  cancelToken,
+}): Promise<unknown> => {
+  const entities = store.getState().entities.entities;
+  const Brand = entities.Brand;
+
+  return defaultEntityBehavior.fetchFks(
+    Brand.path,
+    ['id'],
+    (data: any) => {
```
Comment (fmendozzac) [inline web/portal/platform/src/entities/Brand/SelectOptions.ts:15]: any is forbidden

File: web/portal/platform/src/entities/Currency/ForeignKeyGetter.tsx [hunk 5]
```diff
@@ -0,0 +1,22 @@
+import { ForeignKeyGetterType } from '@irontec/ivoz-ui/entities/EntityInterface';
+import { autoSelectOptions } from '@irontec/ivoz-ui/entities/DefaultEntityBehavior';
+import { CurrencyPropertyList } from './CurrencyProperties';
+
+/** TODO remove this file unless you need to change default behaviour **/
```
Comment (fmendozzac) [inline web/portal/platform/src/entities/Currency/ForeignKeyGetter.tsx:5]: Remove this file

File: web/portal/platform/src/entities/Currency/ForeignKeyResolver.tsx [hunk 6]
```diff
@@ -0,0 +1,22 @@
+import { autoForeignKeyResolver } from '@irontec/ivoz-ui/entities/DefaultEntityBehavior';
+import { foreignKeyResolverType } from '@irontec/ivoz-ui/entities/EntityInterface';
+import { CurrencyPropertiesList } from './CurrencyProperties';
+
+/** TODO remove this file unless you need to change default behaviour **/
```
Comment (fmendozzac) [inline web/portal/platform/src/entities/Currency/ForeignKeyResolver.tsx:5]: Remove this file

File: web/portal/platform/src/entities/Currency/Form.tsx [hunk 7]
```diff
@@ -0,0 +1,29 @@
...
+  const fkChoices = useFkChoices({
+    foreignKeyGetter,
+    entityService,
+    row,
+    match,
+  });
+
+  const groups: Array<FieldsetGroups | false> = [
+    {
+      legend: _('Main'),
```
Comment (fmendozzac) [inline web/portal/platform/src/entities/Currency/Form.tsx:21]: it could go empty

File: web/portal/platform/src/entities/Feature/Form.tsx [hunk 8]
```diff
@@ -0,0 +1,29 @@
...
+  const fkChoices = useFkChoices({
+    foreignKeyGetter,
+    entityService,
+    row,
+    match,
+  });
+
+  const groups: Array<FieldsetGroups | false> = [
+    {
+      legend: _('Main'),
```
Comment (fmendozzac) [inline web/portal/platform/src/entities/Feature/Form.tsx:21]: it could be empty

File: web/portal/platform/src/entities/Feature/SelectOptions.ts [hunk 9]
```diff
@@ -0,0 +1,49 @@
...
+  callback,
+  cancelToken,
+}): Promise<unknown> => {
+  const entities = store.getState().entities.entities;
+  const Feature = entities.Feature;
+
+  return defaultEntityBehavior.fetchFks(
+    Feature.path,
+    ['id', 'iden', 'name'],
+    (data: any) => {
```
Comment (fmendozzac) [inline web/portal/platform/src/entities/Feature/SelectOptions.ts:17]: any is forbidden

File: web/portal/platform/src/entities/Language/Form.tsx [hunk 10]
```diff
@@ -0,0 +1,29 @@
...
+  const fkChoices = useFkChoices({
+    foreignKeyGetter,
+    entityService,
+    row,
+    match,
+  });
+
+  const groups: Array<FieldsetGroups | false> = [
+    {
+      legend: _('Main'),
```
Comment (fmendozzac) [inline web/portal/platform/src/entities/Language/Form.tsx:21]: it could be empty

File: web/portal/platform/src/entities/Feature/Feature.tsx [hunk 11]
```diff
@@ -0,0 +1,37 @@
...
+  },
+};
+
+const Feature: EntityInterface = {
+  ...defaultEntityBehavior,
+  icon: AccountTreeIcon,
+  iden: 'Feature',
+  title: _('Feature', { count: 2 }),
+  path: '/features',
+  toStr: (row: any) => row.name.en,
```
Comment (fmendozzac) [inline web/portal/platform/src/entities/Feature/Feature.tsx:29]: any is forbidden

File: web/portal/platform/src/entities/Language/SelectOptions.ts [hunk 12]
```diff
@@ -0,0 +1,27 @@
...
+  callback,
+  cancelToken,
+}): Promise<unknown> => {
+  const entities = store.getState().entities.entities;
+  const Language = entities.Language;
+
+  return defaultEntityBehavior.fetchFks(
+    Language.path,
+    ['id', 'iden'],
+    (data: any) => {
```
Comment (fmendozzac) [inline web/portal/platform/src/entities/Language/SelectOptions.ts:15]: any is forbidden

File: web/portal/platform/src/entities/Currency/SelectOptions.ts [hunk 13]
```diff
@@ -0,0 +1,29 @@
...
+  callback,
+  cancelToken,
+}): Promise<unknown> => {
+  const entities = store.getState().entities.entities;
+  const Currency = entities.Currency;
+
+  return defaultEntityBehavior.fetchFks(
+    Currency.path,
+    ['id', 'name', 'symbol'],
+    (data: any) => {
```
Comment (fmendozzac) [inline web/portal/platform/src/entities/Currency/SelectOptions.ts:16]: any is forbidden

File: web/portal/platform/src/entities/NotificationTemplate/SelectOptions/FaxSelectOptions.ts [hunk 14]
```diff
@@ -0,0 +1,27 @@
...
+  callback,
+  cancelToken,
+}): Promise<unknown> => {
+  const entities = store.getState().entities.entities;
+  const NotificationTemplate = entities.NotificationTemplate;
+
+  return defaultEntityBehavior.fetchFks(
+    `${NotificationTemplate.path}?type=fax`,
+    ['id', 'name'],
+    (data: any) => {
```
Comment (fmendozzac) [inline web/portal/platform/src/entities/NotificationTemplate/SelectOptions/FaxSelectOptions.ts:15]: any is forbidden

File: web/portal/platform/src/entities/NotificationTemplate/SelectOptions/InvoiceSelectOptions.ts [hunk 15]
```diff
@@ -0,0 +1,27 @@
...
+  callback,
+  cancelToken,
+}): Promise<unknown> => {
+  const entities = store.getState().entities.entities;
+  const NotificationTemplate = entities.NotificationTemplate;
+
+  return defaultEntityBehavior.fetchFks(
+    `${NotificationTemplate.path}?type=invoice`,
+    ['id', 'name'],
+    (data: any) => {
```
Comment (fmendozzac) [inline web/portal/platform/src/entities/NotificationTemplate/SelectOptions/InvoiceSelectOptions.ts:15]: any is forbidden

File: web/portal/platform/src/entities/NotificationTemplate/SelectOptions/InvoiceSelectOptions.ts [hunk 16]
```diff
@@ -0,0 +1,27 @@
...
+      const options: any = {};
```
Comment (fmendozzac) [inline web/portal/platform/src/entities/NotificationTemplate/SelectOptions/InvoiceSelectOptions.ts:16]: any is forbidden

File: web/portal/platform/src/entities/NotificationTemplate/SelectOptions/FaxSelectOptions.ts [hunk 17]
```diff
@@ -0,0 +1,27 @@
...
+      const options: any = {};
```
Comment (fmendozzac) [inline web/portal/platform/src/entities/NotificationTemplate/SelectOptions/FaxSelectOptions.ts:16]: any is forbidden

File: web/portal/platform/src/entities/NotificationTemplate/SelectOptions/LowBalanceSelectOptions.ts [hunk 18]
```diff
@@ -0,0 +1,27 @@
...
+  cancelToken,
+}): Promise<unknown> => {
+  const entities = store.getState().entities.entities;
+  const NotificationTemplate = entities.NotificationTemplate;
+
+  return defaultEntityBehavior.fetchFks(
+    `${NotificationTemplate.path}?type=lowbalance`,
+    ['id', 'name'],
+    (data: any) => {
+      const options: any = {};
```
Comment (fmendozzac) [inline web/portal/platform/src/entities/NotificationTemplate/SelectOptions/LowBalanceSelectOptions.ts:16]: any is forbidden

File: web/portal/platform/src/entities/NotificationTemplate/SelectOptions/MaxDailyUsageSelectOptions.ts [hunk 19]
```diff
@@ -0,0 +1,27 @@
...
+  callback,
+  cancelToken,
+}): Promise<unknown> => {
+  const entities = store.getState().entities.entities;
+  const NotificationTemplate = entities.NotificationTemplate;
+
+  return defaultEntityBehavior.fetchFks(
+    `${NotificationTemplate.path}?type=maxDailyUsage`,
+    ['id', 'name'],
+    (data: any) => {
```
Comment (fmendozzac) [inline web/portal/platform/src/entities/NotificationTemplate/SelectOptions/MaxDailyUsageSelectOptions.ts:15]: any is forbidden

File: web/portal/platform/src/entities/NotificationTemplate/SelectOptions/MaxDailyUsageSelectOptions.ts [hunk 20]
```diff
@@ -0,0 +1,27 @@
...
+      const options: any = {};
```
Comment (fmendozzac) [inline web/portal/platform/src/entities/NotificationTemplate/SelectOptions/MaxDailyUsageSelectOptions.ts:16]: any is forbidden

File: web/portal/platform/src/entities/NotificationTemplate/SelectOptions/SelectOptions.ts [hunk 21]
```diff
@@ -0,0 +1,31 @@
...
+  callback,
+  cancelToken,
+}): Promise<unknown> => {
+  const entities = store.getState().entities.entities;
+  const NotificationTemplate = entities.NotificationTemplate;
+
+  return defaultEntityBehavior.fetchFks(
+    NotificationTemplate.path + '?_order[name]=ASC',
+    ['id', 'name'],
+    (data: any) => {
```
Comment (fmendozzac) [inline web/portal/platform/src/entities/NotificationTemplate/SelectOptions/SelectOptions.ts:16]: any is forbidden

File: web/portal/platform/src/entities/NotificationTemplate/SelectOptions/VoicemailSelectOptions.ts [hunk 22]
```diff
@@ -0,0 +1,27 @@
...
+  callback,
+  cancelToken,
+}): Promise<unknown> => {
+  const entities = store.getState().entities.entities;
+  const NotificationTemplate = entities.NotificationTemplate;
+
+  return defaultEntityBehavior.fetchFks(
+    `${NotificationTemplate.path}?type=voicemail`,
+    ['id', 'name'],
+    (data: any) => {
```
Comment (fmendozzac) [inline web/portal/platform/src/entities/NotificationTemplate/SelectOptions/VoicemailSelectOptions.ts:15]: any is forbidden

File: web/portal/platform/src/entities/NotificationTemplate/SelectOptions/VoicemailSelectOptions.ts [hunk 23]
```diff
@@ -0,0 +1,27 @@
...
+      const options: any = {};
```
Comment (fmendozzac) [inline web/portal/platform/src/entities/NotificationTemplate/SelectOptions/VoicemailSelectOptions.ts:16]: any is forbidden

File: web/portal/platform/src/entities/ProxyTrunk/SelectOptions.ts [hunk 24]
```diff
@@ -0,0 +1,28 @@
...
+  callback,
+  cancelToken,
+}): Promise<unknown> => {
+  const entities = store.getState().entities.entities;
+  const ProxyTrunk = entities.ProxyTrunk;
+
+  return defaultEntityBehavior.fetchFks(
+    ProxyTrunk.path + '?_order[ip]=ASC',
+    ['id', 'ip'],
+    (data: any) => {
```
Comment (fmendozzac) [inline web/portal/platform/src/entities/ProxyTrunk/SelectOptions.ts:16]: any is forbidden

File: web/portal/platform/src/entities/Brand/Field/BrandLogo.tsx [hunk 25]
```diff
@@ -0,0 +1,26 @@
...
+  PropertyCustomFunctionComponentProps,
+} from '@irontec/ivoz-ui';
+import { BrandPropertyList } from '../BrandProperties';
+
+type RouteTypeValues = BrandPropertyList<string>;
+type RouteTypeProps = PropertyCustomFunctionComponent<
+  PropertyCustomFunctionComponentProps<RouteTypeValues>
+>;
+
+const BrandLogo: RouteTypeProps = (props: any): JSX.Element | null => {
```
Comment (fmendozzac) [inline web/portal/platform/src/entities/Brand/Field/BrandLogo.tsx:12]: any is forbidden

File: web/portal/platform/src/entities/Brand/Field/City.tsx [hunk 26]
```diff
@@ -0,0 +1,18 @@
...
+  PropertyCustomFunctionComponentProps,
+} from '@irontec/ivoz-ui';
+import { BrandPropertyList } from '../BrandProperties';
+
+type RouteTypeValues = BrandPropertyList<string>;
+type RouteTypeProps = PropertyCustomFunctionComponent<
+  PropertyCustomFunctionComponentProps<RouteTypeValues>
+>;
+
+const City: RouteTypeProps = (props: any): JSX.Element | null => {
```
Comment (fmendozzac) [inline web/portal/platform/src/entities/Brand/Field/City.tsx:12]: any is forbidden

File: web/portal/platform/src/entities/Brand/Field/Country.tsx [hunk 27]
```diff
@@ -0,0 +1,18 @@
...
+  PropertyCustomFunctionComponentProps,
+} from '@irontec/ivoz-ui';
+import { BrandPropertyList } from '../BrandProperties';
+
+type RouteTypeValues = BrandPropertyList<string>;
+type RouteTypeProps = PropertyCustomFunctionComponent<
+  PropertyCustomFunctionComponentProps<RouteTypeValues>
+>;
+
+const Country: RouteTypeProps = (props: any): JSX.Element | null => {
```
Comment (fmendozzac) [inline web/portal/platform/src/entities/Brand/Field/Country.tsx:12]: any is forbidden

File: web/portal/platform/src/entities/Brand/Field/PostalAddress.tsx [hunk 28]
```diff
@@ -0,0 +1,18 @@
...
+  PropertyCustomFunctionComponentProps,
+} from '@irontec/ivoz-ui';
+import { BrandPropertyList } from '../BrandProperties';
+
+type RouteTypeValues = BrandPropertyList<string>;
+type RouteTypeProps = PropertyCustomFunctionComponent<
+  PropertyCustomFunctionComponentProps<RouteTypeValues>
+>;
+
+const PostalAddress: RouteTypeProps = (props: any): JSX.Element | null => {
```
Comment (fmendozzac) [inline web/portal/platform/src/entities/Brand/Field/PostalAddress.tsx:12]: any is forbidden

File: web/portal/platform/src/entities/Brand/Field/PostalCode.tsx [hunk 29]
```diff
@@ -0,0 +1,18 @@
...
+  PropertyCustomFunctionComponentProps,
+} from '@irontec/ivoz-ui';
+import { BrandPropertyList } from '../BrandProperties';
+
+type RouteTypeValues = BrandPropertyList<string>;
+type RouteTypeProps = PropertyCustomFunctionComponent<
+  PropertyCustomFunctionComponentProps<RouteTypeValues>
+>;
+
+const PostalCode: RouteTypeProps = (props: any): JSX.Element | null => {
```
Comment (fmendozzac) [inline web/portal/platform/src/entities/Brand/Field/PostalCode.tsx:12]: any is forbidden

File: web/portal/platform/src/entities/Brand/Field/RegisterInfo.tsx [hunk 30]
```diff
@@ -0,0 +1,18 @@
...
+  PropertyCustomFunctionComponentProps,
+} from '@irontec/ivoz-ui';
+import { BrandPropertyList } from '../BrandProperties';
+
+type RouteTypeValues = BrandPropertyList<string>;
+type RouteTypeProps = PropertyCustomFunctionComponent<
+  PropertyCustomFunctionComponentProps<RouteTypeValues>
+>;
+
+const RegisterInfo: RouteTypeProps = (props: any): JSX.Element | null => {
```
Comment (fmendozzac) [inline web/portal/platform/src/entities/Brand/Field/RegisterInfo.tsx:12]: any is forbidden

**Review by fmendozzac — APPROVED** [review]

File: web/portal/platform/src/entities/AdministratorRelPublicEntity/SelectOptions.ts [hunk 31]
```diff
@@ -11,11 +12,15 @@ const AdministratorRelPublicEntitySelectOptions: SelectOptionsType = ({
...
-      const options: any = {};
+    ['id', 'administrator'],
+    (data: Array<EntityValues>) => {
+      const options: DropdownChoices = [];
       for (const item of data) {
-        options[item.id] = item.id;
+        options.push({
+          id: item.id as string,
+          label: (item.administrator as { [key: string]: string })
+            .name as string,
```
Comment (mmadariaga) [inline web/portal/platform/src/entities/AdministratorRelPublicEntity/SelectOptions.ts:22]:
```suggestion
            .name,
```

File: web/portal/platform/src/entities/Brand/Brand.tsx [hunk 32]
```diff
@@ -0,0 +1,130 @@
...
+import PostalAddress from './Field/PostalAddress';
+import PostalCode from './Field/PostalCode';
+import Province from './Field/Province';
+import RegisterInfo from './Field/RegisterInfo';
+import Form from './Form';
+
+const properties: BrandProperties = {
+  currency: {
+    label: _('Currency'),
+    required: true,
```
Comment (mmadariaga) [inline web/portal/platform/src/entities/Brand/Brand.tsx:19]: unnecessary

File: web/portal/platform/src/entities/Brand/Brand.tsx [hunk 33]
```diff
@@ -0,0 +1,130 @@
...
+  },
+  defaultTimezone: {
+    label: _('Default timezone'),
+    required: true,
+    default: 145,
+  },
+  domainUsers: {
+    label: _('SIP domain'),
+    required: true,
+    maxLength: 190,
```
Comment (mmadariaga) [inline web/portal/platform/src/entities/Brand/Brand.tsx:29]: unnecessary

File: web/portal/platform/src/entities/Brand/Brand.tsx [hunk 34]
```diff
@@ -0,0 +1,130 @@
...
+  },
+  nif: {
+    label: 'TIN',
+    required: true,
```
Comment (mmadariaga) [inline web/portal/platform/src/entities/Brand/Brand.tsx:33]: unnecessary

File: web/portal/platform/src/entities/Brand/Brand.tsx [hunk 35]
```diff
@@ -0,0 +1,130 @@
...
+    label: _('Language'),
+    default: 1,
+  },
+  logo: {
+    label: _('Logo'),
+    type: 'file',
+  },
+  name: {
+    label: _('Name'),
+    required: true,
```
Comment (mmadariaga) [inline web/portal/platform/src/entities/Brand/Brand.tsx:46]: unnecessary

File: web/portal/platform/src/entities/Brand/Brand.tsx [hunk 36]
```diff
@@ -0,0 +1,130 @@
...
+  },
+  maxCalls: {
+    label: _('Max calls'),
+    default: 2,
+  },
+  features: {
+    label: _('Features', { count: 20 }),
+    null: _('There are not associated elements'),
+    $ref: '#/definitions/Feature',
```
Comment (mmadariaga) [inline web/portal/platform/src/entities/Brand/Brand.tsx:55]: Is this required?

File: web/portal/platform/src/entities/Brand/Brand.tsx [hunk 37]
```diff
@@ -0,0 +1,130 @@
...
+    enum: {
```
Comment (mmadariaga) [inline web/portal/platform/src/entities/Brand/Brand.tsx:56]: featires are array of integers. Why is this a enum?

File: web/portal/platform/src/entities/Brand/Brand.tsx [hunk 38]
```diff
@@ -0,0 +1,130 @@
...
+      Active: _('Active'),
+      feature: _('Feature'),
+      ids: _('Ids'),
+    },
+  },
+  proxyTrunks: {
+    label: _('Proxy Trunks', { count: 20 }),
+    null: _('There are not associated elements'),
+    $ref: '#/definitions/ProxyTrunk',
+    enum: {
```
Comment (mmadariaga) [inline web/portal/platform/src/entities/Brand/Brand.tsx:66]: enum?

File: web/portal/platform/src/entities/Brand/Brand.tsx [hunk 39]
```diff
@@ -0,0 +1,130 @@
...
+      Active: _('Active'),
+      feature: _('Proxy Trunk'),
+      ids: _('Ids'),
+    },
+  },
+  postalAddress: {
+    label: _('Postal Address'),
+    required: true,
+    maxLength: 10,
```
Comment (mmadariaga) [inline web/portal/platform/src/entities/Brand/Brand.tsx:75]: unnecessary & maxLength is wrong

File: web/portal/platform/src/entities/Brand/Field/City.tsx [hunk 40]
```diff
@@ -0,0 +1,18 @@
...
+
+type RouteTypeValues = BrandPropertyList<string>;
+type RouteTypeProps = PropertyCustomFunctionComponent<
+  PropertyCustomFunctionComponentProps<RouteTypeValues>
+>;
+
+const City: RouteTypeProps = (props): JSX.Element | null => {
+  const { values } = props;
+
+  return <>{(values.invoice as { [key: string]: string }).town}</>;
```
Comment (mmadariaga) [inline web/portal/platform/src/entities/Brand/Field/City.tsx:15]:
```suggestion
  return <>{(values.invoice as Record<string, string>).town}</>;
```

**Review by mmadariaga — CHANGES_REQUESTED** [review]

File: web/portal/platform/src/entities/Feature/SelectOptions.ts [hunk 41]
```diff
@@ -0,0 +1,29 @@
...
+  const Feature = entities.Feature;
+
+  return defaultEntityBehavior.fetchFks(
+    Feature.path,
+    ['id', 'iden', 'name'],
+    (data: Array<EntityValues>) => {
+      const options: DropdownChoices = [];
+
+      for (const item of data) {
+        options.push({ id: item.id as string, label: item.iden as string });
```
[the remark this answers is not in the document]
Comment (mmadariaga) [reply web/portal/platform/src/entities/Feature/SelectOptions.ts:20]: I'll continue the review once you've fixed the issues above

File: web/portal/platform/src/entities/Brand/Field/PostalCode.tsx [hunk 42]
```diff
@@ -0,0 +1,18 @@
...
+  PropertyCustomFunctionComponentProps,
+} from '@irontec/ivoz-ui';
+import { BrandPropertyList } from '../BrandProperties';
+
+type RouteTypeValues = BrandPropertyList<string>;
+type RouteTypeProps = PropertyCustomFunctionComponent<
+  PropertyCustomFunctionComponentProps<RouteTypeValues>
+>;
+
+const PostalCode: RouteTypeProps = (props): JSX.Element | null => {
```
Comment (mmadariaga) [inline web/portal/platform/src/entities/Brand/Field/PostalCode.tsx:12]: In every custom component that doesn't return null:

```suggestion
const PostalCode: RouteTypeProps = (props): JSX.Element => {
```

File: web/portal/platform/src/router/useAclFilteredEntityMap.tsx [hunk 43]
```diff
@@ -1,4 +1,4 @@
-import { AboutMe, EntityAcl } from 'store/clientSession/aboutMe';
+import { AboutMe, EntityAcl } from '../store/clientSession/aboutMe';
```
Comment (mmadariaga) [inline web/portal/platform/src/router/useAclFilteredEntityMap.tsx:1]: I don't think this is necessary

**Review by mmadariaga — CHANGES_REQUESTED** [review]

**Review by mmadariaga — APPROVED** [review]

**Pull request merged**